src/LaMEM_io.jl

```diff
@@ -13,6 +13,41 @@ import Base: show, size
 export LaMEM_grid, read_LaMEM_inputfile
 export save_LaMEM_markers_parallel, save_LaMEM_topography
 export get_processor_partitioning, read_data_VTR, read_data_PVTR, create_partitioning_file
+export crop_bounds, get_proc_bound, get_proc_grid, get_particles_distribution, get_LaMEM_grid_info, get_processor_partitioning_info, check_markers_directory, setup_model_domain, LaMEM_partitioning_info
+
+"""
+Structure that holds information about the LaMEM partitioning
+"""
+struct LaMEM_partitioning_info <: AbstractGeneralGrid
```

Comment:
Usually the name of the structs follow the camel convention. Better rename it as `LaMEMPartitioningInfo`